keyboards/4pplet/aekiso60_rev_a/config.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4444
+#define PRODUCT_ID      0x0001
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    4pplet
+#define PRODUCT         AEKISO60 Rev A
+#define DESCRIPTION     A steezy KEEB
+
+/* key matrix size */
+#define MATRIX_ROWS 5
+#define MATRIX_COLS 14
+// ROWS: Top to bottom, COLS: Left to right
+//						  0  1  2  3  4  5  6  7  8  9  a  b  c  d
+#define MATRIX_ROW_PINS {C2,D0,B0,C7,C5}
+#define MATRIX_COL_PINS {C4,C6,B7,B6,B5,B4,B3,B2,B1,D6,D5,D4,D2,D1}
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* define if matrix has ghost */
+//#define MATRIX_HAS_GHOST
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
+
+#define QMK_ESC_OUTPUT 	C4 // usually COL
+#define QMK_ESC_INPUT 	C2 // usually ROW
+
+#define GRAVE_ESC_CTRL_OVERRIDE
+#define GRAVE_ESC_ALT_OVERRIDE
+#define GRAVE_ESC_GUI_OVERRIDE
+#define GRAVE_ESC_SHIFT_OVERRIDE
```

Comment:
> I didn't mark every instance, but the ones I flagged persist across all three keyboards.
> 
> Also, if you're intending to perhaps have multiple revisions of these boards, your directory trees should be `keyboards/4pplet/<keyboard>/<revision>` instead of `keyboards/4pplet/<keyboard>_<revision>` as it is now, then you can easily share common code at the keyboard level, and place code for specific revisions at the revision level.

This is great input, I thought this was not recommended. I have moved revisions into folders, I hope the PCB's will stay relevant and get further revisions. Does the current folder structure look ok? Thanks for all the feedback! 😄 

I have added VIA-keymaps and removed the extra layers from the default-keymaps.